reference/Upload-API.yaml

```diff
@@ -879,5 +879,299 @@ paths:
                       etag: 55a64df551425fcc55e4d42a148795d9f25f89d6
         description: The request body should contain a reference to each part along with the Etag of each upload.
     parameters: []
+  '/api/v1/{apiUserId}/{apiKey}/{targetId}/{languageCode}/search':
+    parameters:
+      - schema:
+          type: string
+        name: apiUserId
+        in: path
+        required: true
+        description: Guid identifying the requesting client
+      - schema:
+          type: string
+        name: apiKey
+        in: path
+        required: true
+        description: Second layer encrypted identifier for the requesting client
+      - schema:
+          type: string
+        name: targetId
+        in: path
+        description: Guid identifying the target where articles are allowed to be published
+      - schema:
+          type: string
+        name: languageCode
+        in: path
+        required: true
+        description: Two characters ISO language code
+    get:
+      summary: Article search
+      tags: []
+      responses:
+        '200':
+          description: |-
+            A sample response
```

Comment:
Can you please add the schema in the responses section and the example separately?
And make the example easy to read as well please, for example, if it's a list of players just showing 2 or 3 entries perhaps?
![Screenshot 2022-07-01 at 08 56 36](https://user-images.githubusercontent.com/8457942/176851981-f2aad3a8-ce5f-42bb-a2e2-02d80ea33b38.png)